boa/src/builtins/regexp/mod.rs

```diff
@@ -289,39 +448,10 @@ impl RegExp {
     // ///
     // /// [spec]: https://tc39.es/ecma262/#sec-get-regexp.prototype.source
     // /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/source
-    // fn get_source(this: &Value, _: &[Value], _: &mut Context) -> Result<Value> {
+    // pub(crate) fn get_source(this: &Value, _: &[Value], _: &mut Context) -> Result<Value> {
```

Comment:
Why this change? Do we need this commented code?

Reply:
I left it so we can uncomment it when we implement `get RegExp.prototype.source`